components/01-atoms/buttons/_buttons.scss

```diff
@@ -0,0 +1,86 @@
+$button-radii: (
+  'soft': var(--button-radius-soft),
+  'pill': 100vw,
+);
+$button-style: (
+  'filled': 1rem,
+  'outline': 2rem,
+);
+$button-outline-weights: (
+  '1': var(--border-thickness-1),
+  '2': var(--border-thickness-2),
+  '4': var(--border-thickness-4),
+);
+
+.button {
+  --color-button-outline: var(--color-link);
+
+  border: var(--border-thickness-button, var(--border-thickness-2)) solid
+    var(--color-button-border);
+  border-radius: var(--border-radius-button, var(--radius-0));
+  background: var(--color-button-bg);
+  color: var(--color-button-text);
+  font-weight: var(--font-weights-mallory-500);
+  padding: var(--size-spacing-4) var(--size-spacing-7);
```

Comment:
add `cursor:pointer;` maybes???

Reply:
Yeah, I know we do that a lot for buttons, but I've also heard that the pointer should be reserved for links (`<a>` tags).

@kara-franco do you have any info on whether there's an accessibility reason to use or not use the pointer cursor for buttons?

Here's a reference, but it doesn't give advice on when to use one or another: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Reply:
I've heard that as well but I've also been under the impression that any element that can be clicked by a user should use a pointer. I'm going to approve this PR but lets wait on merging until @kara-franco chimes in.

Reply:
@kara-franco Thanks for the references! I've shared them internally as well.